app/mailers/failed_registration_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class FailedRegistrationMailer < ActionMailer::Base
+  default from: 'RADenquiries@moneyadviceservice.org.uk'
```

Comment:
Shall we move this values to an env var? What are your thoughts about it?

Reply:
The email address is actually used in many places and to implement it as an env var is a big change which I think would detract from this feature. I will create a TP card to consider your suggestion.